packages/elements/src/slider/index.ts

```diff
@@ -833,18 +884,43 @@ export class Slider extends ControlElement {
     }
 
     const newThumbPosition = this.stepRange !== 0 ? nearestValue : thumbPosition;
-    const displayedValue = this.format(this.getValueFromPosition(newThumbPosition));
+    const value = this.getValueFromPosition(newThumbPosition);
 
+    this.persistChangedData(value);
+  }
+
+  /**
+   * Saves changed data into correct field
+   * @param value value of changed data
+   * @returns {void}
+   */
+  private persistChangedData (value: number): void {
+    const newValue = this.format(value).toString();
     if (this.range) {
       if (this.changedThumb === this.thumbs[0]) {
-        this.from = this.validateFrom(Number(displayedValue)).toString();
+        this.from = this.validateRange(newValue, SliderDataName.from);
       }
       else {
-        this.to = this.validateTo(Number(displayedValue)).toString();
+        this.to = this.validateRange(newValue, SliderDataName.to);
       }
     }
     else {
-      this.value = displayedValue;
+      this.value = newValue;
+    }
+  }
+
+  /**
+   * Validate from or to value and re-asssign
+   * @param value from or to value
+   * @param type from or to
+   * @returns validated from or to
+   */
+  private validateRange (value: string, type: SliderDataName): string {
```

Comment:
Maybe this function needs better comment. I am not sure what does it try to validate

Reply:
It was a wrapper function of `validateFrom` and `validateTo`. I've decided to remove it and improve code comment.